scripts/schema/config/boundaries.json

```diff
@@ -219,6 +219,22 @@
         }
       }
     },
+    "Periodic":
+    {
+      "type": "array",
+      "additionalItems": false,
+      "items":
+      {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["DonorAttributes", "ReceiverAttributes"],
+        "properties":
+        {
+          "DonorAttributes": { "$ref": "#/$defs/DonorAttributes" },
+          "ReceiverAttributes": { "$ref": "#/$defs/ReceiverAttributes" }
+        }
+      }
+    },
```

Comment:
I don't believe this is correct: running `./scripts/validate-config examples/cube/cube_periodic_xy_user.json` from the root folder gives an error. This should also be updated once the interface is in terms of pairs of attributes `[[1,2], [3,4], [5,6]]` for instance in the case of the cube.